Added hidden filter to data streams tab

yuliacech opened the pull request:
## Summary

Fixes [#83709](https://github.com/elastic/kibana/issues/83709).

This PR adds support for hidden data streams in Index Management app. A new view button allows user to toggle several filters for data streams. Currently, there are 'Fleet-managed data streams' and 'hidden data streams'. Fleet-managed data streams are enabled by default, hidden data streams are disabled by default. 
The code reuses view filter button created for index template list.

### Screenshots

There are some data streams, but all are filtered out: table is displayed with no rows.
<img width="1024" alt="Screenshot 2020-12-08 at 18 59 10" src="https://user-images.githubusercontent.com/6585477/101621754-774a9480-3a16-11eb-9e0e-bbf1dee9bb36.png">

Hidden data streams with a label
<img width="1032" alt="Screenshot 2020-12-08 at 18 59 19" src="https://user-images.githubusercontent.com/6585477/101621910-a82ac980-3a16-11eb-87f2-dc1f242460dc.png">

When there are no data streams at all, the empty prompt is shown. This UI has not changed 
<img width="981" alt="Screenshot 2020-12-09 at 12 11 56" src="https://user-images.githubusercontent.com/6585477/101622762-c1804580-3a17-11eb-92a9-817cb2f40f46.png">

### How to test
A hidden data stream is usually already preexisting `ilm-history-5`. To create more hidden data streams: 
``` 
PUT _index_template/test
{
  "template": {},
  "index_patterns": [
    "test"
  ],
  "data_stream": {
    "hidden": true   
  }
}

POST test/_doc
{
  "@timestamp": "2020-01-27"
}
```

### Checklist
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios

## Release note
In Index Management app, hidden data streams are now displayed and indicated by a label.

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/)
* Commit: 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a
* [Pipeline Steps](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/flowGraphTable) (look for red circles / failed steps)
* [Interpreting CI Failures](https://www.elastic.co/guide/en/kibana/current/interpreting-ci-failures.html)

### Failed CI Steps
* [Build X-Pack Plugins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/execution/node/125/log)
* [Report failed tests, if necessary](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/execution/node/137/log)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ‼️ ERROR: metrics for 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a were not reported

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/)
* Commit: 3cd3321debdeb5d9b713a4bbed484523b9fd1967
* This comment will update when the build is complete

### Failed CI Steps
* [X-Pack Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/execution/node/341/log)

### History
* :broken_heart: [Build #91996](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/) failed 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"}]}
PIPELINE-->

yuliacech commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/)
* Commit: ea7380c45d6cb569c02c4018b4b10fd053573009
* This comment will update when the build is complete

### Failed CI Steps
* [X-Pack Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/execution/node/339/log)

### History
* :broken_heart: [Build #92006](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/) failed 3cd3321debdeb5d9b713a4bbed484523b9fd1967
* :broken_heart: [Build #91996](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/) failed 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/)
* Commit: 6629910a0504500d65574ec6bab5d97d4480e25c

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/446390d86a5e74b020c2a00d745f664ac200bb38) | [after](https://github.com/elastic/kibana/commit/6629910a0504500d65574ec6bab5d97d4480e25c) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 523 | 525 | +2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/446390d86a5e74b020c2a00d745f664ac200bb38) | [after](https://github.com/elastic/kibana/commit/6629910a0504500d65574ec6bab5d97d4480e25c) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 1.5MB | 1.5MB | +1.9KB |

#### Distributable file count
| id | [before](https://github.com/elastic/kibana/commit/446390d86a5e74b020c2a00d745f664ac200bb38) | [after](https://github.com/elastic/kibana/commit/6629910a0504500d65574ec6bab5d97d4480e25c) | diff |
| --- | --- | --- | --- |
| `default` | 46920 | 47680 | +760 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/446390d86a5e74b020c2a00d745f664ac200bb38) | [after](https://github.com/elastic/kibana/commit/6629910a0504500d65574ec6bab5d97d4480e25c) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 124.1KB | 124.1KB | +45.0B |

### History
* :broken_heart: [Build #92290](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/) failed ea7380c45d6cb569c02c4018b4b10fd053573009
* :broken_heart: [Build #92006](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/) failed 3cd3321debdeb5d9b713a4bbed484523b9fd1967
* :broken_heart: [Build #91996](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/) failed 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/","number":"92396","commit":"6629910a0504500d65574ec6bab5d97d4480e25c"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

alisonelizabeth reviewed (commented):
Nice work @yuliacech! This is looking great. I left a few comments in the code. I think we might be able to make the api integration tests a little more robust; let me know what you think.

I'd also like to test the deep-linking before I approve. Can you remind me on how/where to test that? I checked the index templates UI, but from what I could see we are only indicating if there is an associated data stream and not actually linking to the data stream.

I also had a few UX questions/suggestions (some of which might be better suited for a separate PR):
- I have trouble understanding the difference between `managed`, `system` and `hidden` 😄. Should `hidden` actually be `managed`, or is it a different behavior for data streams? For example, the ILM index template is actually marked as `managed` in the UI, but the associated data stream is `hidden`.
- Could we also add the badge to the details panel of the data stream? You can check out index templates as an example.
<img width="626" alt="Screen Shot 2020-12-09 at 10 39 59 AM" src="https://user-images.githubusercontent.com/5226211/101654931-0bb0f900-3a0f-11eb-991b-c5370f4143e1.png">

- For `system` templates, we add an additional warning when deleting. Would this be applicable to hidden data streams?
<img width="804" alt="Screen Shot 2020-12-09 at 11 24 52 AM" src="https://user-images.githubusercontent.com/5226211/101656896-4156e180-3a11-11eb-9f80-b1bfed86f8e3.png">

yuliacech commented:
@elasticmachine merge upstream

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/)
* Commit: 7475605c258c84493d55828741b40e9a06453474

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/cb29438b0d23ea770fd8fb766cb8b3c7f44d2a03) | [after](https://github.com/elastic/kibana/commit/7475605c258c84493d55828741b40e9a06453474) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 523 | 525 | +2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/cb29438b0d23ea770fd8fb766cb8b3c7f44d2a03) | [after](https://github.com/elastic/kibana/commit/7475605c258c84493d55828741b40e9a06453474) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 1.5MB | 1.5MB | +1.9KB |

#### Distributable file count
| id | [before](https://github.com/elastic/kibana/commit/cb29438b0d23ea770fd8fb766cb8b3c7f44d2a03) | [after](https://github.com/elastic/kibana/commit/7475605c258c84493d55828741b40e9a06453474) | diff |
| --- | --- | --- | --- |
| `default` | 47010 | 47770 | +760 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/cb29438b0d23ea770fd8fb766cb8b3c7f44d2a03) | [after](https://github.com/elastic/kibana/commit/7475605c258c84493d55828741b40e9a06453474) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 124.1KB | 124.1KB | +45.0B |

### History
* :green_heart: [Build #92396](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/) succeeded 6629910a0504500d65574ec6bab5d97d4480e25c
* :broken_heart: [Build #92290](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/) failed ea7380c45d6cb569c02c4018b4b10fd053573009
* :broken_heart: [Build #92006](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/) failed 3cd3321debdeb5d9b713a4bbed484523b9fd1967
* :broken_heart: [Build #91996](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/) failed 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/","number":"92396","commit":"6629910a0504500d65574ec6bab5d97d4480e25c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/","number":"93475","commit":"7475605c258c84493d55828741b40e9a06453474"}]}
PIPELINE-->

yuliacech commented:
Hi @alisonelizabeth , thank you so much for your review! I addressed the feedback, would you mind having another look? 
- I think 'managed' data streams will usually not be 'hidden', but if that happens I put the badges into a badge group, so that they would stack on a smaller screen. WDYT? 
<img width="297" alt="Screenshot 2020-12-10 at 19 17 01" src="https://user-images.githubusercontent.com/6585477/101812933-58362a80-3b1c-11eb-958f-d1dcf238fbdc.png">
<img width="189" alt="Screenshot 2020-12-10 at 19 17 10" src="https://user-images.githubusercontent.com/6585477/101813758-73556a00-3b1d-11eb-9f28-c1b21ed6b47e.png">

- Added the badges to the detail flyout as well
<img width="508" alt="Screenshot 2020-12-10 at 19 19 19" src="https://user-images.githubusercontent.com/6585477/101813211-b82cd100-3b1c-11eb-9a9a-b5c126566880.png">

- As far as I understand, hidden data stream can be deleted, because this is just a property of a data stream set via an index template. I would not add any special information there, as there is currently no semantics attached to this property. 

- I see how that can be confusing, but managed templates are not connected to managed data streams. Those are data streams managed by Fleet. Do you think we should change 'managed' label to 'Fleet-managed' or similar? 

- Thanks for your notes about the api integration tests, I updated them to be more resilient against ES changes. 

- To check cross-linking, you could open Indices list, enable hidden indices and click on data stream name 'ilm-history-5'. I also added a query param, should we need to cross-link to data streams tab with hidden filter on with `includeHidden=true`.

yuliacech commented on x-pack/plugins/index_management/server/routes/api/data_streams/register_get_route.ts:
```diff
@@ -65,12 +65,24 @@ const enhanceDataStreams = ({
   });
 };
 
+const getDataStreams = (client: ElasticsearchClient, name = '') => {
```
I think the code would not be much easier with a ternary operator for `path`, so I'd prefer to have a default value. Or did you mean something else?

alisonelizabeth replied:
My main concern is the path ends up resolving to `/_data_stream//` is there is no name specified

yuliacech replied:
Thanks for bringing this up, @alisonelizabeth ! I haven't thought about it and that's a valid concern. I changed the empty string to a wildcard `*` and deleted the closing forward slash. So if name isn't defined, the path would be `/_data_stream/*`.

yuliacech commented on x-pack/plugins/index_management/server/routes/api/data_streams/register_get_route.ts:
```diff
@@ -65,12 +65,24 @@ const enhanceDataStreams = ({
 };
 
+const getDataStreams = (client: ElasticsearchClient, name = '') => {
+  // TODO update when elasticsearch client has update requestParams for 'indices.getDataStream'
```
There is no issue as elasticsearch-js client generates their typings for API automatically. I think we just need to wait for the next client update, so I left this comment. But let me know what you think :)

yuliacech commented on x-pack/plugins/index_management/public/application/lib/data_streams.tsx:
```diff
@@ -6,10 +6,34 @@
 
-export const isManagedByIngestManager = (dataStream: DataStream): boolean => {
+export const isFleetManaged = (dataStream: DataStream): boolean => {
+  // TODO check if the wording will change to 'fleet'
```
I'll remove this as soon as I get info from the Fleet team :)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/)
* Commit: 8d13d447438a62726190f41994b5dca269caf3c7
* This comment will update when the build is complete

### Failed CI Steps
* [X-Pack Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/execution/node/333/log)

### History
* :green_heart: [Build #93475](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/) succeeded 7475605c258c84493d55828741b40e9a06453474
* :green_heart: [Build #92396](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/) succeeded 6629910a0504500d65574ec6bab5d97d4480e25c
* :broken_heart: [Build #92290](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/) failed ea7380c45d6cb569c02c4018b4b10fd053573009
* :broken_heart: [Build #92006](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/) failed 3cd3321debdeb5d9b713a4bbed484523b9fd1967
* :broken_heart: [Build #91996](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/) failed 9afa24b80e1e3b7dfb3b009ba8206d599a2e601a

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/91996/","number":"91996","commit":"9afa24b80e1e3b7dfb3b009ba8206d599a2e601a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/","number":"92396","commit":"6629910a0504500d65574ec6bab5d97d4480e25c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/","number":"93475","commit":"7475605c258c84493d55828741b40e9a06453474"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/","number":"93611","commit":"8d13d447438a62726190f41994b5dca269caf3c7"}]}
PIPELINE-->

alisonelizabeth reviewed (approved):
Thanks for addressing my feedback @yuliacech! I think the CITs need to be updated to react to your most recent changes, but otherwise LGTM. Tested again and verified the cross-linking.

Also, the badge on the details panel is misaligned for me - possibly caused by using the `EuiBadgeGroup`?

<img width="430" alt="Screen Shot 2020-12-10 at 2 23 02 PM" src="https://user-images.githubusercontent.com/5226211/101819615-7e93a000-3af3-11eb-888b-2f769dc85af7.png">

alisonelizabeth commented:
> I think 'managed' data streams will usually not be 'hidden', but if that happens I put the badges into a badge group, so that they would stack on a smaller screen. WDYT?

👍 

> As far as I understand, hidden data stream can be deleted, because this is just a property of a data stream set via an index template. I would not add any special information there, as there is currently no semantics attached to this property.

👍 

> I see how that can be confusing, but managed templates are not connected to managed data streams. Those are data streams managed by Fleet. Do you think we should change 'managed' label to 'Fleet-managed' or similar?

I'm not as familiar with this area, but if they are always managed by Fleet I think it would be helpful to update the label to be more specific. As a future enhancement, I think it could also be helpful to provide a tooltip or something to each badge that gives a better explanation of what it means.

yuliacech commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93948/)
* Commit: 865b781acaaf6a113c4a60117bcbd8b2cf3cea11
* This comment will update when the build is complete

### Failed CI Steps
* [X-Pack Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93948/execution/node/329/log)

### History
* :broken_heart: [Build #93611](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/) failed 8d13d447438a62726190f41994b5dca269caf3c7
* :green_heart: [Build #93475](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/) succeeded 7475605c258c84493d55828741b40e9a06453474
* :green_heart: [Build #92396](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/) succeeded 6629910a0504500d65574ec6bab5d97d4480e25c
* :broken_heart: [Build #92290](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/) failed ea7380c45d6cb569c02c4018b4b10fd053573009
* :broken_heart: [Build #92006](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/) failed 3cd3321debdeb5d9b713a4bbed484523b9fd1967

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92006/","number":"92006","commit":"3cd3321debdeb5d9b713a4bbed484523b9fd1967"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/","number":"92396","commit":"6629910a0504500d65574ec6bab5d97d4480e25c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/","number":"93475","commit":"7475605c258c84493d55828741b40e9a06453474"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/","number":"93611","commit":"8d13d447438a62726190f41994b5dca269caf3c7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93948/","number":"93948","commit":"865b781acaaf6a113c4a60117bcbd8b2cf3cea11"}]}
PIPELINE-->

yuliacech commented:
Thanks for re-reviewing @alisonelizabeth ! I fixed the alignment issue in the detail panel :) Will create a separate PR for wording of Fleet-managed data streams and a possible tooltip for the label.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93993/)
* Commit: 01eb4ffeec4ae3b8cad79a7104bf1079937f10fe

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/3f7f0be2a0c806b24c4816decefdfe9cbb5d5b23) | [after](https://github.com/elastic/kibana/commit/01eb4ffeec4ae3b8cad79a7104bf1079937f10fe) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 523 | 525 | +2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/3f7f0be2a0c806b24c4816decefdfe9cbb5d5b23) | [after](https://github.com/elastic/kibana/commit/01eb4ffeec4ae3b8cad79a7104bf1079937f10fe) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 1.5MB | 1.5MB | +2.0KB |

#### Distributable file count
| id | [before](https://github.com/elastic/kibana/commit/3f7f0be2a0c806b24c4816decefdfe9cbb5d5b23) | [after](https://github.com/elastic/kibana/commit/01eb4ffeec4ae3b8cad79a7104bf1079937f10fe) | diff |
| --- | --- | --- | --- |
| `default` | 47129 | 47889 | +760 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/3f7f0be2a0c806b24c4816decefdfe9cbb5d5b23) | [after](https://github.com/elastic/kibana/commit/01eb4ffeec4ae3b8cad79a7104bf1079937f10fe) | diff |
| --- | --- | --- | --- |
| `indexManagement` | 124.1KB | 124.1KB | +45.0B |

### History
* :broken_heart: [Build #93948](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93948/) failed 865b781acaaf6a113c4a60117bcbd8b2cf3cea11
* :broken_heart: [Build #93611](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/) failed 8d13d447438a62726190f41994b5dca269caf3c7
* :green_heart: [Build #93475](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/) succeeded 7475605c258c84493d55828741b40e9a06453474
* :green_heart: [Build #92396](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/) succeeded 6629910a0504500d65574ec6bab5d97d4480e25c
* :broken_heart: [Build #92290](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/) failed ea7380c45d6cb569c02c4018b4b10fd053573009

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92290/","number":"92290","commit":"ea7380c45d6cb569c02c4018b4b10fd053573009"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/92396/","number":"92396","commit":"6629910a0504500d65574ec6bab5d97d4480e25c"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93475/","number":"93475","commit":"7475605c258c84493d55828741b40e9a06453474"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93611/","number":"93611","commit":"8d13d447438a62726190f41994b5dca269caf3c7"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93948/","number":"93948","commit":"865b781acaaf6a113c4a60117bcbd8b2cf3cea11"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/93993/","number":"93993","commit":"01eb4ffeec4ae3b8cad79a7104bf1079937f10fe"}]}
PIPELINE-->
